src/components/photoGallery/PhotoGallery.scss

```
@@ -0,0 +1,14 @@
+.imageGallery {
```

Class names should be written in `kebab-case`:
```suggestion
.image-gallery {
```